packages/react-native-windows-init/src/Cli.ts

```diff
@@ -124,6 +124,13 @@ export const windowsInitOptions = initOptions({
     default: undefined, // This must be undefined because we define the conflicts field below. Defining a default here will break the version option
     conflicts: 'version',
   },
+  template: {
+    type: 'string',
+    describe: '[Experimental] Specify a project template to use',
+    hidden: true,
```

Comment:
Not necessarily a problem with **this** PR, but I think having experimental options set with `hidden: true` makes it hard to get them correct since they don't appear in the help. Honestly I think the `[Experimental]` warning is enough to tell people to beware.